src/framework/audio/internal/audioengine.cpp

```diff
@@ -151,7 +136,10 @@ void AudioEngine::setBuffer(IAudioBufferPtr buffer)
 {
     m_buffer = buffer;
     m_buffer->setSource(m_mixer);
-    m_worker->setAudioBuffer(m_buffer);
+
+    //! TODO It doesn't look obvious.
+    //! Requires detailed research
+    //m_worker->setAudioBuffer(m_buffer);
```

Comment:
I see like we're initializing audio buffer in modulesetup. So I'm wondering how often we'll need to call AudioEngine::setBuffer. For me it seems like we'll need it just to setup it at the very beginning (at the initialization step) and that's all. Am I missing something?

Reply:
In generally you right. 
Only maybe we can set buffer for write to file or any other destination, but I'm not sure at the moment.